notifications_python_client/errors.py

```diff
@@ -5,26 +5,34 @@
 from builtins import super
 from future import standard_library
 standard_library.install_aliases()
+
 REQUEST_ERROR_STATUS_CODE = 503
 REQUEST_ERROR_MESSAGE = "Request failed"
 
+TOKEN_ERROR_GENERIC_MESSAGE = "Invalid token: see https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header"  # noqa
```

Comment:
Maybe the message could be something like: 
"Invalid token. For token specifications see https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header" ?

@karlchillmaid what do you think?

Reply:
Oh yes, good point. I'd almost forgotten to say I'd definitely like your review on the content please @karlchillmaid

Reply:
Would this be too long?

`Invalid token. Find out how to create the authorisation header: https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header`

I'm just trying to get a bit of continuity between the link text and the page/heading users land on.

Reply:
I think length is fine but I might suggest something in the middle ground:

`Invalid token. Find out how to create a valid JSON Web token: https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header`

As the issue isn't that they aren't sending an authorisation header, it's that the token which is the value of that header is not a valid JWT token.

Reply:
@idavidmcdonald Yeah, that's a more helpful message. It specifically addresses the problem they're trying to solve. Given the fact we don't want the message to get too long, we can forget about flagging the 'Authorisation header' heading in advance.

Reply:
Sorry, I didn't quite understand your last sentence. Are you saying that we should drop `#authorisation-header` from the provided link?